docs/rancher.md

```diff
@@ -12,9 +12,20 @@ sidebar_label: Rancher
 ### Installation of OpenEBS
 
 - Use Rancher dashboard and enable the helm stable from Catalogs.
+
+![Helm-Enable](docs/assets/rancher_enable_helm.PNG)
+
+
 - Launch **openebs** from Catalog Apps.
+
+![openebs-installation](docs/assets/rancher_openebs_install.PNG)
+
+
 - Add storage class. In storage class provide details such as Name, Provisoner and Parameters.
```

Comment:
6. Click **Storage** from the Rancher dashboard.

<<screen>>

7. Select **Storage Class** to add storage class. Add details such as Name, Provisioner, and Parameters.